### PR TITLE
Update turek

### LINKED ISSUE #113: Writing drag or lift values does not occur at the end
When using a monitor in Fluent to write data such as lift and drag coefficients to a file, the writing operation is done at the start of a time step rather than the end, after `(ti-menu-load-string "solve dual-time-iterate 1 0").` In this way, the data always lags a time step. Moreover, as the nodes are displaced before this data is printed, it is not a pure 'lagging', but there is also some deviation:
(left is written to the file and right is taken from the gui (correct))
```
"drag-rset"
"Time Step" "flow-time etc.."
("Time Step" "flow-time" "drag")
0 0 0
1 0.002 0				|	0.028120664
2 0.004 0.02812066304860059		|	0.086690928
3 0.006 0.08669090634451442		|	0.15015176
4 0.008 0.1501516958477103		|	0.21807432
5 0.01 0.2180741961229031		|	0.28941945


"lift-rset"
"Time Step" "flow-time etc.."
("Time Step" "flow-time" "lift")
0 0 0
1 0.002 0				|	-0.0011475312
2 0.004 -0.001147531168200983		|	-0.002402138
3 0.006 -0.002402137926860938		|	-0.0022637938
4 0.008 -0.002263793670246033		|	-0.0019021652
5 0.01 -0.001902165000144555		|	-0.0015391751
```

This excerpt from the log file clearly shows what is happening:

 ```
RECEIVED MESSAGE continue

### timestep 2, iteration 1 ###
solve dual-time-iterate 1 0
Updating solution at time level N...
 done.

Updating mesh at time level N... 
Finished UDF move_nodes.

Finished UDF move_nodes.

Finished UDF move_nodes.

Finished UDF move_nodes.

Finished UDF move_nodes.

Finished UDF move_nodes.
done.

 iter  continuity  x-velocity  y-velocity     time/iter
!  450 solution is converged
   450  5.0626e-07  2.9174e-09  1.7646e-09  0:00:00    0
  step  flow-time        lift   flow-time        drag        cl-1        cd-1
     2  4.0000e-03 -1.1475e-03  4.0000e-03  2.8121e-02 -1.1475e-03  2.8121e-02
Flow time = 0.004s, time step = 2
solve iterate 1000
  iter  continuity  x-velocity  y-velocity     time/iter
!  450 solution is converged
   450  5.0626e-07  2.9174e-09  1.7646e-09  0:00:31 1000
   460  5.6188e-01  4.5879e-04  2.6474e-04  0:00:31  990
```
The deviation is rather small at this time, but I expect it will be much larger is the deformation are bigger.

The monitor was defined as follows:
```
report reference-values velocity 1
report reference-values area 1
report reference-values density 2
solve monitors force drag-coefficient y circleoutside beamtopoutside beamrightoutside beambottomoutside () y y drag.frp n n 1 0
solve monitors force lift-coefficient y circleoutside beamtopoutside beamrightoutside beambottomoutside () y y lift.frp n n 0 1

```

### PR DESCRIPTION
**Pull request by nicolasdlss**
**Description** Updates the Turek test examples (finer mesh). A parameter field is added to the fluent solver wrapper to allow the execution of certain journal commands at the end of each time step.

**Users' experience affected?** NA

**Other parts of the code affected?** NA

**Tests** The Turek test example works, both steady and unsteady (FSI2). The Fluent tests have been verified to still work.

**Related issues** Closes #113 

**Checklist**
- [x] Style guidelines
    1. CoCoNuT follows [the PEP8 style guide](https://www.python.org/dev/peps/pep-0008/).
    2. Comments should be lowercase.
    3. Errors should start with capitals, preferably without punctuation unless it's multiple sentences.
    4. Strings should use single quotes whenever possible.	
- [x] Self-review of code performed
- [x] Code has been commented (particularly in hard-to-understand areas)
- [x] Documentation was updated correspondingly
- [x] New and existing unit tests pass locally with changes

### CODE REVIEW

File: examples/tube_fluent2d_abaqus2d/parameters.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
           ],
           "cores": 4,
           "fluent_gui": false,
-          "max_nodes_per_face": 2,
+          "max_nodes_per_face": 4,
```
Comment (hdolfen) [inline examples/tube_fluent2d_abaqus2d/parameters.json:73]: Can you please verify this again.

File: examples/tube_fluent2d_abaqus2d_steady/parameters.json [hunk 2]
```diff
@@ -70,7 +70,7 @@
           ],
           "cores": 4,
           "fluent_gui": false,
-          "max_nodes_per_face": 2,
+          "max_nodes_per_face": 4,
```
Comment (hdolfen) [inline examples/tube_fluent2d_abaqus2d_steady/parameters.json:73]: Can you please verify again?

File: examples/turek_fluent2d_abaqus2d/parameters.json [hunk 3]
```diff
@@ -97,11 +98,11 @@
               ]
             }
           ],
-          "cores": 4,
+          "cores": 8,
           "fluent_gui": false,
-          "max_nodes_per_face": 2,
-          "flow_iterations": 1000,
-          "save_iterations": 200
+          "max_nodes_per_face": 3,
```
Comment (hdolfen) [inline examples/turek_fluent2d_abaqus2d/parameters.json:103]: Can you please verify this again?

File: examples/turek_fluent2d_abaqus2d_steady/parameters.json [hunk 4]
```diff
@@ -96,9 +97,9 @@
               ]
             }
           ],
-          "cores": 4,
+          "cores": 8,
           "fluent_gui": false,
-          "max_nodes_per_face": 2,
+          "max_nodes_per_face": 3,
```
Comment (hdolfen) [inline examples/turek_fluent2d_abaqus2d_steady/parameters.json:102]: Please verify again.

**Review by hdolfen — APPROVED** [review]
Everything works fine, but I think that the number of nodes per face was already correct before, so the number should not be enlarged.

**Review by NPynaert — APPROVED** [review]
turek_fluent2d_abaqus2d is tested and works fine.

File: examples/tube_fluent2d_abaqus2d/parameters.json (hunk 1 above)
Comment (nicolasdlss) [reply examples/tube_fluent2d_abaqus2d/parameters.json:73]: Fixed.

File: examples/turek_fluent2d_abaqus2d/parameters.json (hunk 3 above)
Comment (nicolasdlss) [reply examples/turek_fluent2d_abaqus2d/parameters.json:103]: Fixed.

File: examples/turek_fluent2d_abaqus2d_steady/parameters.json (hunk 4 above)
Comment (nicolasdlss) [reply examples/turek_fluent2d_abaqus2d_steady/parameters.json:102]: Fixed.

**Comment by nicolasdlss** [comment]
Fixed the number of nodes per face in all occurrences and made the corresponding documentation a bit more clear.

**Pull request merged**